terraform/weblogic_variables.tf

```diff
@@ -197,3 +197,29 @@ variable "deploy_sample_app" {
   default     = true
 }
 
+variable "configure_secure_mode" {
+  type        = bool
+  description = "Set to true to configure a secure WebLogic domain"
+  default     = true
```

Comment:
I don't think that the default should be for secured production mode to be selected. Please check with Adrian Padilla.